### PR TITLE
Kk/cherry-pick

### PR DESCRIPTION
**Pull request by jaspnas**
Cherry picking a bunch of stuff

### CODE REVIEW

**Review by Nyygard — APPROVED** [review]
Lgtm

File: events/urls.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from django.urls import re_path, path
 
+from django.urls import re_path
+
```
Comment (theoabw) [inline events/urls.py:4]: Double re_path import?

**Review by Theoabw — CHANGES_REQUESTED** [review]

**Comment by theoabw** [comment]
Also don't know if it's intended or not, but it doesn't pass kk-test at least for me

**Comment by theoabw** [comment]
image

**Comment by theoabw** [comment]
image

**Comment by jaspnas** [comment]
Pls test on QA @Nyygard @Theoabw

**Pull request merged**